### ISSUE TITLE
[feature] Require email for refferal reward

### ISSUE THREAD

**Issue by DWAA1660** [Ctrlpanel-gg/panel#706]
# 🚀 Feature Request

### Is your proposal related to a problem?

*Provide a clear and concise description of what the problem is.
For example, "I can't change x and it makes me have to do manual work"*

I cannot easily verify emails are real users or just random characters people fill in when they get reward

### Describe the solution you'd like

*Provide a clear and concise description of what you want to happen.
For Example, "Automate the changing of x so I don't have to do it manually"*

Maybe have it so if require email verification is set to true users don't get referral reward until user verifies there email address so at-least its more difficult to bypass.
### Additional context

*Is there anything else you can add about the proposal?
You might want to link to related issues here, if you haven't already.*

Maybe also add so you can tell what users a user referred when looking at them so its easier to stop abuse of it.

**Comment by GeckoBoy84** [comment]
Should be optional via a checklist for email and/or discord or none

**Comment by S0ly** [comment]
can someone explain me this issue ?

**Comment by LakshmiBhaskarPVL** [comment]
@S0ly  so the user wants us to add a feature where we have email verification for redeeming referrals
he wants to verify if the emails are real and are not just random emails like "shitahhperson@nig.com"

**Comment by DWAA1660** [comment]
I think what I requested with this is that the person that referred a user don't get rewarded until said referral verifies his email to prevent alte

**Comment by S0ly** [comment]
yeah I get that but its not something that is already present ? @MrWeez

**Comment by simbabimba-dev** [comment]
This has been worked on, PR will be released shortly.

**Comment by MrWeez** [comment]
Implemented. Will be available in next release
